hangman.py

```diff
@@ -0,0 +1,129 @@
+import random
+from words import words
+
+def hangman_pick_word():
+    """
+    Randomly picks a theme from the words.py file and converts to upper case.
+    """
+    theme = random.choice(list(words.keys()))
+    word = random.choice(words.get(theme)).upper()
+    return theme, word
+
+def hangman_generate_hint(theme, word):
```

Comment:
Perhaps name this `hangman_print_hint` to make it consistent with it's functionality?

Reply:
Yeah that is a good point! Changed!